tests/30rooms/06invite.pl

```diff
@@ -107,3 +111,424 @@
          Future->done(1);
       });
    };
+
+test "Can invite existing 3pid",
+   requires => [qw( user more_users test_http_server_hostandport await_http_request )],
```

Comment:
Headsup warning: at some point relatively soon, I'll have `await_http_request` out as a real function rather than having to drag it around the env. all the time.